libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/MapboxNavigationAccounts.kt

```diff
@@ -46,9 +46,8 @@ class MapboxNavigationAccounts private constructor() : SkuTokenProvider {
         }
     }
 
-    override fun obtainSkuToken(resourceUrl: String?, querySize: Int): String {
+    override fun obtainSkuToken(resourceUrl: String?, querySize: Int): String? {
```

Comment:
I believe we don't need to make it nullable as `generateToken` is always to return a non-null empty `String`. Removing the `check` should be enough.